context/app/static/js/components/publications/PublicationsPanelList/hooks.ts

```diff
@@ -0,0 +1,55 @@
+import useSearchData from 'js/hooks/useSearchData';
+import { fetcher } from 'js/helpers/swr';
+import { SearchRequest, SearchResponse } from '@elastic/elasticsearch/lib/api/types';
+import { buildPublicationPanelProps, PublicationHit } from './utils';
+
+const getPublicationsByStatusQuery: (publicationStatus: string) => SearchRequest = (publicationStatus) => ({
+  query: {
+    bool: {
+      must: [
+        {
+          term: {
+            'entity_type.keyword': 'Publication',
+          },
+        },
+        {
+          term: {
```

Comment:
Is publicationStatus always one value or can it be multiple?

Reply:
This query is specific to one publication status at a time based on the currently selected tab on the page (publication or preprint).

Reply:
Great, thanks. In that case, the query looks good to me!